test/parallel/test-trace-events-api.js

```diff
@@ -0,0 +1,161 @@
+// Flags: --expose-gc --no-warnings
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasTracing)
+  common.skip('missing trace events');
+
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+const {
+  createTracing,
+  getEnabledCategories,
+  getEnabledTracingObjects
+} = require('trace_events');
+
+const isChild = process.argv[2] === 'child';
+const enabledCategories = isChild ? 'foo' : undefined;
+
+assert.strictEqual(getEnabledCategories(), enabledCategories);
+[1, 'foo', true, false, null, undefined].forEach((i) => {
```

Comment:
nit: might be good to deal with this (i.e. better error message). It might be worth noting that non-`string` values get coerced to a string in the docs. Okay, if it gets done in a follow-on PR.
```js
> t.createTracing({categories: [Symbol("foo")]})
TypeError: Cannot convert a Symbol value to a string
    at new Tracing (trace_events.js:25:21)
    at Object.createTracing (trace_events.js:75:10
```